uint8 vel_level
float32 orientation